package.json

```diff
@@ -52,7 +52,7 @@
     "extract-api": "api-extractor run --local --verbose"
   },
   "devDependencies": {
-    "@microsoft/api-extractor": "^7.40.1",
+    "@microsoft/api-extractor": "7.43.0",
```

Comment:
should this be "^7.43.0"? or is the pinning intentional

Reply:
Yeah it's related to #118